std.go

```diff
@@ -3,9 +3,13 @@ package logger
 import (
 	"context"
 	"log"
+	"sync"
 )
 
-type stdErrLogger struct{}
+type stdErrLogger struct {
+	mu         sync.Mutex
+	attributes map[string]any
+}
```

Comment:
I am thinking we should finish the implementation and print out the attributes in `std()` and `stdf()`.

If we don't think we should spend the time on it, then I would convert `AddRequestAttribute()` and `RemoveRequestAttributes()` methods into no-op methods and remove `attributes` from the struct.

Reply:
I just realized that I was confused again when I left this comment. The Attributes are for the parent logger, and in this case, we explicitly do not have a parent logger.

Reply:
Correct... I waffled back and forth on this, at one point having them implemented as no-ops. I landed on implementing them in case things changed to use/need them in the future. However, if that ever occurs, then their implementation can simply be updated at them time. I'll change them to no-ops 👍.